contents.json

```diff
@@ -12013,6 +12013,32 @@
       ],
       "updated": "2024-11-14 15:28:06 UTC"
     },
+    {
+      "title": "Element X",
+      "category-ids": [
+        "communication"
+      ],
+      "tags": [
+        "swift",
+        "iphone",
+        "ipad",
+        "matrix",
+        "rust"
+      ],
+      "description": "Secure decentralised chat/VoIP, newer version of Element",
+      "source": "https://github.com/element-hq/element-x-ios",
+      "homepage": "https://element.io",
+      "license": "agpl-3.0",
+      "itunes": "https://apps.apple.com/us/app/element-x-secure-chat-call/id1631335820",
```

Comment:
```suggestion
      "itunes": "https://apps.apple.com/app/element-x-secure-chat-call/id1631335820",
```

Reply:
I rebased to allow the branch to be a single commit that uses the suggested title format